src/components/web-components/rank-card/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { StyleSheet } from 'react-native';
+import { ITheme } from '../../../core/theme/itheme';
+
+export default (theme: ITheme) =>
+    StyleSheet.create({
+        container: {
+            backgroundColor: theme.colors.rankCard,
+            marginRight: '16.84%',
+            marginLeft: '16.84%',
+            marginTop: 37,
```

Comment:
same here something like BASE_DIMENSION * 8 
or you can make BASE_DIMENSION higher since the values are pretty big